aws/resource_aws_vpc_endpoint_service.go

```diff
@@ -79,6 +79,31 @@ func resourceAwsVpcEndpointService() *schema.Resource {
 			"private_dns_name": {
 				Type:     schema.TypeString,
 				Computed: true,
+				Optional: true,
+			},
+			"private_dns_name_config": {
```

Comment:
Nit: To match the [API naming convention](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ServiceConfiguration.html) this should probably be `private_dns_name_configuration`

```suggestion
			"private_dns_name_configuration": {
```